src/ecdsa/ecdsa.py

```diff
@@ -467,3 +467,15 @@ def point_is_valid(generator, x, y):
 generator_brainpoolp512r1 = ellipticcurve.PointJacobi(
     curve_brainpoolp512r1, _Gx, _Gy, 1, _q, generator=True
 )
+
+# Weierstass Curve 25519
+_a = 0x2aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa984914a144
+_b = 0x7b425ed097b425ed097b425ed097b425ed097b425ed097b4260b5e9c7710c864
+_p = 0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffed
+_Gx = 0x2aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaad245a
+_Gy = 0x20ae19a1b8a086b4e01edd2c7748d14c923d4d7e6d7c61b229e9c5a27eced3d9
+_r = 0x1000000000000000000000000000000014def9dea2f79cd65812631a5cf5d3ed
+
+curve_25519_weier = ellipticcurve.CurveFp(_p, _a, _b, 1)
+generator_25519_weier = ellipticcurve.PointJacobi(
+    curve_25519_weier, _Gx, _Gy, 1, _r, generator=True)
```

Comment:
nit: newline missing
nit: closing paren needs to be on next line, `black` will complain otherwise